docs/notebooks/thomson.ipynb

```diff
@@ -0,0 +1,298 @@
+    "# 1064 nm are used (532 corresponds to a frequency-doubled probe beam from such a laser).\n",
+    "probe_wavelength = 532*u.nm\n",
+    "\n",
+    "# Array of wavelengths over which to calcualte the spectral distribution\n",
+    "wavelengths = np.arange(probe_wavelength.value-60, probe_wavelength.value+60, 0.01)*u.nm\n",
+    "\n",
+    "# The scattering geometry is defined by unit vectors for the orientation of the probe laser beam (probe_n) and\n",
+    "# the path from the scattering volume (where the measurement is made) to the detector (scatter_n).\n",
+    "# These can be setup for any experimental geometry.\n",
+    "probe_vec = np.array([1, 0, 0])\n",
+    "scattering_angle = np.deg2rad(63)\n",
+    "scatter_vec = np.array([np.cos(scattering_angle), np.sin(scattering_angle), 0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In order to calcluate the scattered spectrum, we must also include some information about the plasma. For this plot we'll allow the fract, ion_species, fluid_vel, and ion_vel keywords to keep their default values, describing a single-species H+ plasma at rest in the laboratory frame. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx-thumbnail": {
+     "tooltip": "Thomson Scattering"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "ne = 2e17*u.cm**-3\n",
+    "Te = 12*u.eV\n",
+    "Ti = 10*u.eV\n",
+    "\n",
+    "# This warning filter catches an ImplicitUnitConversionWarning that results from specifying\n",
+    "# temperatures in eV instead of Kelvin. \n",
+    "with warnings.catch_warnings():\n",
+    "    warnings.simplefilter(\"ignore\", ImplicitUnitConversionWarning)\n",
+    "    \n",
+    "    alpha, Skw = thomson.spectral_density(wavelengths, probe_wavelength,\n",
+    "                                          ne, Te, Ti, probe_vec=probe_vec, \n",
+    "                                          scatter_vec=scatter_vec)\n",
+    "\n",
+    "fig, ax = plt.subplots()\n",
+    "ax.plot(wavelengths, Skw, lw=2)\n",
+    "ax.set_xlim(probe_wavelength.value-10, probe_wavelength.value+10)\n",
+    "ax.set_ylim(0, 1e-13)\n",
+    "ax.set_xlabel('$\\lambda$ (nm)')\n",
+    "ax.set_ylabel('S(k,w)')\n",
+    "ax.set_title('Thomson Scattering Spectral Density')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Example Cases in Different Scattering Regimes\n",
+    "\n",
+    "We will now consider several example cases in different scattering regimes. In order to facilitate this, we'll set up each example as a dictionary of plasma parameters:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A single-species and stationary hydrogen plasma with densities and temperatures that mean scattering is dominated by single-electron scattering (rather than collective multi-particle scattering)"
```

Comment:
```suggestion
    "A single-species, stationary hydrogen plasma with a density and temperature that results in the mean scattering being dominated by scattering off of single electrons."
```

The descriptions for each regime need to be cleaned-up a bit so they're easier to read.